test/uk/gov/hmrc/cdsreimbursementclaimfrontend/journeys/SecuritiesJourneyTestData.scala

```diff
@@ -85,15 +86,22 @@ trait SecuritiesJourneyTestData extends JourneyTestData {
       .mapWhenDefined(contactAddress)(_.submitContactAddress _)
       .flatMapEach(reclaims.map(_._1).distinct, (journey: SecuritiesJourney) => journey.selectSecurityDepositId(_))
       .flatMapEach(
-        reclaims.groupBy(_._1).mapValues(_.map { case (_, tc, amount) => (tc, amount) }).toSeq,
+        reclaims
```

Comment:
We use scalafmt to autoformat, why this change?

Reply:
I will check my intellij settings... I thought it detected scalafmt settings

Reply:
sbt compile is reformatting this